laCour/slack-night-mode
# apply theme color to description text of channel

Resolves #140

It appears this class was completely new or never styled previously.

Test Plan:
- Create a new channel, giving it a lengthy description
- In the channel, view the description at the top of the message log
- Verify that the text color is the default light for the theme
